### DISCUSSION TITLE
Launching Fluent in Pre/Post Only-mode

### DISCUSSION THREAD

**Discussion by antonpersson-ba** [ansys/pyfluent#1477] [Q&A] [answered]
Hi, is it possible to launch Fluent via PyFluent in Pre/Post Only-mode if you intend to just do post processing, thus avoiding the need to check out a solver license? This is available in the Fluent GUI launcher under general options.

**Comment by seanpearsonuk** [comment, marked as answer]
@antonpersson-ba you can inject any arguments into `launch_fluent` via the `additional_arguments` argument: https://fluent.docs.pyansys.com/release/0.13/user_guide/launching_ansys_fluent.html.

**Reply by Jojo2587** [reply]
this link isn't working

**Reply by seanpearsonuk** [reply]
You should be able to find the equivalent page in current the user guide(?)

https://fluent.docs.pyansys.com/version/stable/api/launcher/launcher.html
https://fluent.docs.pyansys.com/version/stable/user_guide/session/launching_ansys_fluent.html

**Reply by Jojo2587** [reply]
Those links display the general functions of the launcher command line yes. But there is nothing specific about an way to open a prepost. and additionnal_arguments = "-pp" doesn't work

**Comment by seanpearsonuk** [comment]
Hello @Jojo2587 

`-post`

https://github.com/ansys/pyfluent/issues/3852 (+ linked PR)
